client/src/common/type.ts

```diff
@@ -0,0 +1,9 @@
+export type ModalPortalProps = {
+  children: React.ReactNode;
+};
+export type ModalFrameProps = {
+  leftButtonText: string;
+  rightButtonText: string;
+  children: React.ReactNode;
+  setOnModal: (state: boolean) => void;
+};
```

Comment:
여기도 마찬가지로 이렇게 표현할 수 있겠네요
```suggestion
export type ModalFrameProps = PropsWithChildren<{
  leftButtonText: string;
  rightButtonText: string;
  setOnModal: (state: boolean) => void;
}>;
```